app/grandchallenge/components/models.py

```diff
@@ -1143,6 +1150,14 @@ def _clean_store_in_database(self):
                 f"Interface {self.kind} objects cannot be stored in the database"
             )
 
+    def _clean_example_value(self):
+
+        if self.example_value:
+            civ = ComponentInterfaceValue(
+                interface=self, value=self.example_value
```

Comment:
I don't think that this works with types that are not stored in the database.

Reply:
Checked and updated